calculator-app/src/App.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import './App.css';
+import Keypad from "./Components/Keypad"
+import DisplayResult from "./Components/DisplayResult"
+ 
+class App extends React.Component {
+ constructor(){
+   super()
+   this.state ={
+     displayValue: "0", //this is whats going to dispkay
+     previousValue:"",
+     //curent value- going to be whats previous value , handle the operations
+     operation:"",
+    }
+   }
+   
+   //add new piece of state -boolean if waiting for new val ; if just calculated then do somehting else  
+//    - displayValue (What the screen currently displays)
+// - previousValue (What value the user selected before pressing an operation button)
+// - operation (What operation the user is performing (+/-/÷/x))
+ 
+    handleClick =(e)=>{
+      
+      if(this.state.displayValue === '0'){
+        this.setState({displayValue:e.target.value})
+      } else {
+
+        let displayVal = this.state.displayValue
+        displayVal += e.target.value
+        this.setState({displayValue:displayVal})
+      }
+    }
+
+
+
+
+
+    
+
+
+    handleOperation=(e)=>{ 
+
+        this.setState({previousValue: this.state.displayValue, operation: e.target.value, displayValue:""});  
```

Comment:
If we want to keep the display showing until they've entered the next number, this logic will likely need to change a little.